pyproject.toml

```diff
@@ -33,7 +33,7 @@ classifiers = [
 python = "^3.8, <4.0"
 numpy = ">=1.18.0"
 matplotlib = ">=3.5.0"
-mabwiser = { version = ">=2.7.0", optional = true }
+mabwiser = { version = ">=2.7.1", optional = true }

```

Comment:
@N-Wouda  just checking if we need to update the ALNS version somewhere --and I don't know how the PyPI thing works for ALNS to reflect the change

Reply:
I'll bump versions now and release to PyPI, no worries!